pontoon/contributors/templates/contributors/profile.html

```diff
@@ -97,6 +104,17 @@ <h2 class="display-name">{{ contributor.first_name }}</h2>
                     <a class="button" href="{{ url('pontoon.contributors.settings') }}">Change Settings</a>
                 </div>
                 {% endif %}
+                {% set is_admin = (user.is_authenticated and (user.is_staff or user.is_superuser)) %}
+                    {% if is_admin and not is_my_profile %}
+                    <form id="account-status-form" action="{{ url('pontoon.contributors.toggle_user_status', username=contributor.username) }}" method="POST">
+                        {% csrf_token %}
+                        {% if contributor.is_active %}
+                        <button class="button" type="submit"><i class="icon fa fa-lock fa-xs"></i>Suspend</button>
```

Comment:
For consistency with the title under the avatar, let's use Disable / Enable instead of Suspend / Unsuspend.